omniflixhub/chain.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "omniflixhub",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "OmniFlix Hub",
+    "chain_id": "omniflixhub-1",
+    "daemon_name": "omniflixhubd",
+    "node_home": "$HOME/.omniflixhub",
+    "bech32_prefix": "omniflix",
```

Comment:
Can we please register this to SLIP 173 as well?
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Completed https://github.com/satoshilabs/slips/pull/1359